get/opensearch.go

```diff
@@ -106,3 +113,18 @@ func (cmd *openSearchCmd) getClusterHealth(clusterHealth storage.OpenSearchClust
 
 	return worstStatus
 }
+
+func (cmd *openSearchCmd) printSnapshotBucket(writer io.Writer, ctx context.Context, client *api.Client, openSearch *storage.OpenSearch) error {
+	bucket := &storage.ObjectsBucket{}
+	if err := client.Get(ctx, types.NamespacedName{Name: openSearch.Status.AtProvider.SnapshotsBucket.Name, Namespace: client.Project}, bucket); err != nil {
```

Comment:
It might make sense to verify if `openSearch.Status.AtProvider.SnapshotsBucket.Name` is not an empty string first?